pkgs/games/fteqw/default.nix

```
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchsvn
+, gzip
+, libvorbis
+, libmad
+, SDL2
+, SDL2_mixer
+, libpng
+, alsa-lib
+, gnutls
+, zlib
+, libjpeg
+, vulkan-loader
+, vulkan-headers
+, speex
+, libopus
+, xorg
+, libGL
+}@attrs:
+{
+  fteqw = import ./generic.nix (rec {
+    pname = "fteqw";
+    releaseFile = pname;
+
+    buildFlags = [ "m-rel" ];
+
+    nativeBuildInputs = [
+      vulkan-headers
+    ];
+
+    buildInputs = [
+      gzip libvorbis libmad SDL2 SDL2_mixer
+      libpng alsa-lib gnutls libjpeg vulkan-loader
+      speex libopus xorg.libXrandr xorg.libXcursor
```

```suggestion
      gzip
      libvorbis
      libmad
      SDL2
      SDL2_mixer
      libpng
      alsa-lib
      gnutls
      libjpeg
      vulkan-loader
      speex
      libopus
      xorg.libXrandr
      xorg.libXcursor
```
for better diffs we usually do this